spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each {|f| require f}
 
 RSpec.configure do |config|
-  config.treat_symbols_as_metadata_keys_with_true_values = true
+  config.expect_with(:rspec) { |c| c.syntax = :should }
```

Comment:
This looks like an RSpec 3+ feature. Could you update the gemspec to require a certain major version of RSpec? (A specific major and minor version would be fine as well)